deploy/kustomize/operator/kustomization.yaml

```diff
@@ -21,10 +24,10 @@ patchesJSON6902:
   path: operator-events-namespace-patch.yaml
 
 vars:
-- name: SERVICE_ACCOUNT_NAMESPACE
+- name: OPERATOR_NAMESPACE
   objref:
-    apiVersion: v1
-    kind: ServiceAccount
+    apiVersion: apps/v1
+    kind: Deployment
```

Comment:
The ServiceAccount does not exist when kustomize generates `deploy/kustomize/olm-bundle`. The Deployment references the same namespace and always exists, therefore it gets used.